corehq/apps/domain/decorators.py

```diff
@@ -319,7 +331,8 @@ def _inner(request, domain, *args, **kwargs):
             domain_name, domain = load_domain(request, domain)
             if not domain:
                 raise Http404()
-            if not request.couch_user.is_domain_admin(domain_name):
+            if not _page_is_whitelist(request.path, domain_name) and \
```

Comment:
@czue 
I checked this and when I try to go to the page from whitelist, I getting 403 error. I tried this for all users, this working as previously but when we enable restriction for stuff users then pages from whitelist are available for these users.